application.js

```diff
@@ -22,13 +31,17 @@ app.use(express.static(__dirname + '/public'));
 // Note: important that this is added just before your own Routes
 app.use(mbaasExpress.fhmiddleware());
 
-app.use('/cloud', mbaasExpress.cloud(require('./main.js')));
+app.use('/jbpm', proxy(jbpmHost + ':' + jbpmPort, {
+  decorateRequest: function(proxyReq) {
+    if(jbpmUsername && jbpmPassword)
```

Comment:
just a small style thing, but I'd add braces

See also https://github.com/fheng/best_practice/blob/master/nodejs/code_style.md - may want to consider adding an `.eslintrc.json` to the repo (and possibly remove jshint, since we've sort of grown less fond of it over the years)